mers/releasenotes.md

```diff
@@ -31,21 +30,15 @@ surveillance, and product placement.
 
 The release includes
 
-  * Clear Linux* OS `31410`
-  * x264 version `3759fcb7b48037a5169715ab89f80a0ab4801cdf`
-  * SVT-HEVC `1.4.1`
-  * SVT-AV1 `0.7.0`
-  * FFmpeg `4.1.4`
-  * GStreamer `1.16.0`
-  * OPENCV `4.1.2`
-  * OpenVINO (DLDT) `2019_R2`
-  * GStreamer Video Analytics `0.6.1`
-
-> **Note:**
-     Clear Linux is a rolling distro and by default the auto updater is
-     enabled; if container is alive for some time, the auto updater may update
-     the version that comes by default with the docker Media Stack image. The
-     last validate version for the stack is `31410`.
+  * Ubuntu* OS `20.04 LTS (Focal Fossa)`
+  * x264 version `1771b556ee45207f8711744ccbd5d42a3949b14c`
+  * SVT-HEVC `ead6fdf7c9ff84511b42fc1658c1654b84d83e4b`
+  * SVT-AV1 `v0.8.4`
+  * FFmpeg `7800cc6e82068c6dfb5af53817f03dfda794c568`
+  * GStreamer `1.18.0`
+  * OPENCV `4.4.0`
+  * OpenVINO (DLDT) `2020.4`
```

Comment:
I think you have to specify which component, in this case the model server.

Reply:
We are, and in this case MeRS is not using OVMS, we are using DLDT.

Reply:
Gotcha, shouldn't it be Openvino toolkit?

Reply:
Yes, a.k.a. _Deep Learning Deployment Toolkit_ or DLDT